.github/.linkspector.yml

```diff
@@ -19,5 +19,6 @@ ignorePatterns:
   - pattern: "code.visualstudio.com"
   - pattern: "www.emacswiki.org"
   - pattern: "linux.die.net/man"
+  - pattern: "www.gnu.org"
```

Comment:
I wonder why we need to ignore this.

Reply:
that site probably blocks however linkspector is checking responses, so even though the site is there, the check fails: <https://github.com/coder/coder/actions/runs/12834052725/job/35790477857#step:5:240>